Makefile

```diff
@@ -21,8 +21,9 @@ test:
 
 testsubmodule:
 	cd security/advancedtls && go test -cpu 1,4 -timeout 7m google.golang.org/grpc/security/advancedtls/...
+	cd security/advancedtls && go vet -all ./...
```

Comment:
I believe nobody really uses this `Makefile` anymore, but we must keep it for awhile for backward compatibility with ancient interop test infrastructure that requires it; this change should be unnecessary.